go/enclave/crypto/transaction_blob_crypto.go

```diff
@@ -50,8 +48,28 @@ func NewTransactionBlobCryptoImpl() TransactionBlobCrypto {
 	}
 }
 
+func (t TransactionBlobCryptoImpl) ToExtRollup(r *core.Rollup) common.ExtRollup {
```

Comment:
These are not new, I just hoisted them up the file (since the methods that were above them are no longer public).